Runtime/AvatarCreator/Utils/AvatarRenderHelper.cs

```diff
@@ -11,7 +11,7 @@ public static class AvatarRenderHelper
         public static async Task<Texture2D> GetPortrait(string avatarId, CancellationToken token = default)
         {
             var webRequestDispatcher = new WebRequestDispatcher();
-            var response = await webRequestDispatcher.SendRequest<ResponseTexture>(AvatarEndpoints.GetRenderEndpoint(avatarId), HttpMethod.GET,
+            var response = await webRequestDispatcher.SendRequest<ResponseTexture>($"{Env.RPM_MODELS_BASE_URL}/{avatarId}.png", HttpMethod.GET,
```

Comment:
A small improvement I'd now make (but haven't done here yet) is to move this into the main avatar api class, since this class isn't a helper, it's just a class with another avatar api request in it.